base.rb

```diff
@@ -1,35 +1,35 @@
-def do_something
+def do_something(p)
   
 end
 
-def do_something2(a, b)
+def do_something2(a, b, c, d = 3)
   
 end
 
-def do_something3(a: b)
+def do_something3(c: b)
```

Comment:
__[development][method_definition_validator]__
* The following keyword parameters are not expected. But received. [a] at [use.rb#L3](/pocke/sandbox2/blob/8a8d5fb8801448f1c2d3377c102ddc461a555471/use.rb#L3)